### ISSUE TITLE
Firefox on iOS not able to sync with SyncServer Docker and desktop browser session.

### ISSUE THREAD

**Issue by xthursdayx** [mozilla-services/syncserver#198]
I am running SyncServer in a docker container using Mozilla's image and the standard settings, and connected via an NGINX reverse proxy. My desktop Firefox browser (on MacOS) seems to work fine with the updated `tokenServerURI`, but when I add the same URL (`https://sync.my.server/token/1.0/sync/1.5`) in the Custom Sync Token Server space om Firefox for iOS it doesn't seem to work. It seems to connect fine, and once I sign back in to sync I receive a message on my desktop telling me that my computer is now connected to my phone, but then the Firefox on the phone never syncs again and I am unable to see any synced tabs from my desktop Firefox. 

I'm not sure how to go about troubleshooting this. Any suggestions? Thanks!

**Comment by xthursdayx** [comment]
Has anyone experienced this issue?

**Comment by Kuret** [comment]
Yeah, running in a docker container, except for me everything EXCEPT tabs syncing works fine. (The only feature I really need.). Reverted to Mozilla's servers for now.
